common/source/docs/common-generators.rst

```diff
@@ -17,7 +17,8 @@ The following are all currently supported within ArduPilot:
     RichenPower generator <common-richenpower-generator>
     Intelligent Energy 650W/800W Hydrogen Fuel Cell <common-ie650-fuelcell>
     Intelligent Energy 2.4kW Hydrogen Fuel Cell <common-ie24-fuelcell>
-    Shang Fei Aviation <http://www.svffi.com/en> 
+    Shang Fei Aviation <http://www.svffi.com/en>
+    Loweheiser generator <common-loweheiser-generator>
```

Comment:
not your problem but this list should be alphabetical...if you don't fix, I will